### DISCUSSION TITLE
Disabling Clip

### DISCUSSION THREAD

**Discussion by squideye-net** [dlemstra/Magick.NET#859] [Help] [answered]
I can enable the clipping path with the Clip() function, but how do I disable it? I've tried passing null and empty string, but these throw exceptions.  And passing "#0" as the path argument seems to yield no result.

**Comment by dlemstra** [comment, marked as answer]
You will need to call `RemoveWriteMask`. The intellisense documentation should probably be updated to make this more clear. I can add this myself but feel free to open up a PR for this.

**Reply by squideye-net** [reply]
Thank you very much!
